common/djangoapps/student/models.py

```diff
@@ -1611,7 +1612,16 @@ def refund_cutoff_date(self):
             attribute = self.attributes.filter(namespace='order', name='order_number').last()
 
         order_number = attribute.value
-        order = ecommerce_api_client(self.user).orders(order_number).get()
+        try:
+            order = ecommerce_api_client(self.user).orders(order_number).get()
+        except HttpClientError:
+            log.warning(
+                u"Unable to get details from ecommerce of order={number} for user={user}".format(
+                    number=order_number,
+                    user=self.user.id)
```

Comment:
Can we update the error message to be like `Encountered HttpClientError while getting order details from ecommerce. Order={number}...`